
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title>Kourosh Zarringhalam</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <meta name="description" content="Small Corporation" />
    <meta name="keywords" content="small, corporation" />
    <link rel="stylesheet" href="style.css" type="text/css" />
  </head>
  <body>
    <div id="content">
      <div id="top">
        <p><img src="images/umb-logo-large.gif" alt="Car" hight=65px width=65px/></p>
      </div>
      
      <div id="logo">
	<h1><a href="http://www.umb.edu">University of Massachusetts Boston</a></h1>
	<p><a href="http://www.math.umb.edu">Department of Mathematics</a></p>
      </div>
      
      <ul id="menu">
	<li><a class="current" href="index.html">Home</a></li>
	<!--<li><a href="Teaching.html">Teaching</a></li>-->
	<li><a href="Publications.html">Publications</a></li>
	<li><a href="Software.html">Software</a></li>
	 <li><a href="Contact.html">Contact</a></li>
      </ul>
      
      <div id="mainLong">
	<h3>Publications</h3>
	      <h4><a href="https://scholar.google.com/citations?hl=en&user=ZF7_WeYAAAAJ">Google Scholar Page</a></h4>
	       <div id="line"></div>
	<ol type="1">
		<li>Deep learning trained on H&E tumor ROIs predicts HER2 status and Trastuzumab treatment response in HER2+ breast cancer, <b>2021, BioRxive (in review)</b>. </li> 
        <ul> Saman Farahmand, Aileen I Fernandez, Fahad Shabbir Ahmed, David L. Rimm,  Jeffrey H. Chuang, Emily Reisenbichler, and <b>Kourosh Zarringhalam</b> </ul>

	  	 <li>The extracellular milieu of Toxoplasma-s lytic cycle drives lab-adaptation and promotes changes in lipid metabolism primarily driven by transcriptional reprogramming, <b>2021, BioRxive (in review)</b>. </li> 
        <ul> Vincent A Primo Jr., Yasaman Rezvani, Andrew Farrell, Amir Vajdi Hoojghan, Gabor T Marth, <b>Kourosh Zarringhalam</b>, and Marc-Jan Gubbels </ul>

		 <li>The modular circuitry of apicomplexan cell division plasticity, <b>2021, Frontiers in Cellular and Infection Microbiology </b>. </li> 
        <ul>  Marc-Jan Gubbels, Isabelle Coppens, <b> Kourosh Zarringhalam</b>, Manoj Duraisingh, and Klemens Engelberg. </ul>

	 <li>Susceptibility-associated genetic variation in NEDD9 contributes to prostate cancer initiation and progression, <b>2021, Cancer Research</b>. </li> 
        <ul> Dong Han, Jude N. Owiredu, Bridget M. Healy, Muqing Li, Maryam Labaf, Jocelyn S. Steinfeld, Susan Patalano, Shuai Gao, Mingyu Liu, Jill A. Macoska, <b>Kourosh Zarringhalam</b>, Kellee R. Siegfried, Xin Yuan, Timothy R. Rebbeck and Changmeng Cai. </ul>

	  	  	  <li>Deep learning-based cross-classifications reveal conserved spatial behaviors within tumor histological images, <b>2020, Nature Communications</b>. </li>
        <ul> Javad Noorbakhsh, Saman Farahmand, Ali Foroughi pour, Sandeep Namburi, Dennis Caruana, David Rimm, Mohammad Soltanieh-ha, <b>Kourosh Zarringhalam</b>, Jeffrey H Chuang. </ul>

	  	  <li>ModEx: A text mining system for extracting mode of regulation of Transcription Factor-gene regulatory interaction, <b>2019, Journal of Biomedical Informatics</b>. </li>
        <ul> Saman Farahmand, Todd Riley, <b> Kourosh Zarringhalam </b>. </ul>

	  <li>Causal Inference Engine: A platform for directional gene set enrichment analysis and inference of active transcriptional regulators, <b>2019, Nucleic Acids Research</b>. </li>
        <ul> Saman Farahmand, Corey O'Connor, Jill A. Macoska, <b> Kourosh Zarringhalam </b>. </ul>

	  <li> Patch-DCA: Improved Protein Interface Prediction by utilizing Structural Information and Clustering DCA scores, <b>2019, Bioinformatics</b>. </li>
        <ul> Amir Vajdi, <b> Kourosh Zarringhalam </b>,  Nurit Haspel. </ul>

	  <li>A Free Energy Based Approach For Distance Metric Learning, <b>2019, ACM SIGKDD</b>, Anchorage, AK, USA. </li>
        <ul> Sho Inaba, Carl T. Fakhry, Rahul V. Kulkarni, <b> Kourosh Zarringhalam </b> </ul>

       <li>Clustering on Sparse Data in Non-Overlapping Feature Space with Applications to Cancer Subtyping, <b>2018, IEEE ICDM</b>, Singapore. </li>
        <ul> Tianyu Kang*,<b> Kourosh Zarringhalam </b>, Marieke Kuijjer, Ping Chen, John Quackenbush, and Wei Ding </ul>
	<li>Stochastic Modeling of Gene Regulation by Noncoding Small RNAs in the Strong Interaction Limit, <b>2018, Biophys J.</b> </li>
	<ul>  N. Kumar, <b> K. Zarringhalam </b>, R. Kulkarni </ul>
	<li>Robust phenotype prediction from gene expression data using differential shrinkage of co-regulated genes, <b>2018, Scientific Reports</b>.</li>
	<ul>  <b> K. Zarringhalam </b>, D. Degras, C. Brockel, D. Ziemek </ul>

	<li>A biological network-based regularized artificial neural network model for robust phenotype prediction from gene expression data, <b>2017, BMC Bioinformatics</b>.</li>
	<ul> T. Kang, W. Ding, L. Zhang, D. Ziemek, <b> K. Zarringhalam </b> </ul>

	<li> Bioinformatic Approach for Prediction of CsrA/RsmA-Regulating Small RNAs in Bacteria, <b>2018, Methods, Mol. Biol</b>. Book Chapter</li>
	<ul> C. Fakhry,  <b> K. Zarringhalam </b>, R. Kulkarni </ul>
	<li>Prediction of Bacterial small RNAs in the RsmA (CsrA) and ToxT pathways: a machine learning approach, <b>2017, BMC Genomics</b>.</li>
	<ul> C. Fakhry, P. Kulkarni, P. Chen, R. Kulkarni, <b> K. Zarringhalam </b> </ul>

	
	<li>Identification of competing endogenous rnas (cernas) of the tumor suppressor gene pten: A probabilistic approach, <b>2017, Scientific Reports</b>.</li>
	<ul> <b> K. Zarringhalam </b>, Y. Tay, P. Kulkarni, A. Bester, P. P. Pandolfi, R. Kulkarni </ul>
	
	<li>Frequency modulation of stochastic gene expression bursts by strongly interacting small RNAs, <b>2016, Phys. Rev. E</b>.</li>
	<ul> N. Kumar, T. Jia, <b> K. Zarringhalam </b>, R. Kulkarni </ul>
	
	
	<li>Pressure dependency of localization degree in heavy fermion CeIn3: A density functional theory analysis, <b>2016, Scientifc Reports</b>.</li>
        <ul> Yazdani-Kachoei M, Jalali-Asadabadi S, Ahmad I <b> K. Zarringhalam </b> </ul>
	
	
	<li>Interpreting transcriptional changes using causal graphs: New methods and their practical utility on public networks, <b>2016, BMC Bioinformatics</b>.</li>
        <ul> C. T. Fakhry, P. Choudhary, A. Gutteridge, B. Sidders, P. Chen, D. Ziemek, <b> K. Zarringhalam </b></ul>
	
	
	<li>Biomedical Relation Extraction Using Stochastic Difference Equations, <b>2015, IEEE HPEC</b>.</li>
        <ul> C. T. Fakhry, <b> K. Zarringhalam </b>, P. Chen </ul>
	
	<li>Robust Clinical Outcome Prediction based on Bayesian Analysis of Transcriptional Profiles and Prior Causal Networks, <b>2014, Bioinformatics</b>.</li>
        <ul> <b> K. Zarringhalam </b>, A. Enayetallah, P. Reddy, D. Ziemek </ul>
	
	<li>Molecular causes of transcriptional response: A Bayesian prior knowledge approach, <b>2013, Bioinformatics</b>.</li>
        <ul> <b> K. Zarringhalam </b>, A. Enayetallah, A. Gutteridge, B. Sidders, D. Ziemek </ul>
	
	<li>Dynamics of the ethanolamine glycerophospholipid remodeling
	  network, <b>2012, PLoS ONE</b>.</li>
        <ul> L. Zhang*, N. Diaz–Diaz*, <b>K. Zarringhalam*</b>, 
          M. Hermansson, P. Somerharju, J.
          Chuang. ( * co-first authors )</ul>
	<li>Integrating chemical footprinting data into RNA secondary
	  structure prediction, <b>2012, PLoS ONE</b>.</li>
        <ul> <b>K. Zarringhalam</b>, M. Meyer, I. Dotu, J. Chuang, and P. Clote.</ul>
	<li>Statistical analysis of processes controlling choline and
	  ethanolamine glycerophospholipids molecular species composition, <b>2012, PLoS
	  ONE</b>.</li> 
        <ul> <b>K. Zarringhalam*</b>, L. Zhang*, M. Kiebish, K. Yang, X. Han, R.
          Gross, and J. Chuang. (* co-first authors)</ul>
	<li>An open system for automatic home-cage behavioral analysis and
	  its application to male and female mouse models of Huntington’s
	  disease, <b>2012, Behavioural Brain Research</b>.</li> 
        <ul> <b> K. Zarringhalam</b>, M. Ka, Y. Kook, J. Terranova, Y. Suh,
          O. King. and M. Um. </ul>
	<li>Sequential Adaptive Compressed Sampling Via Huffman Codes, <b>2011, 
	  Sampling Theory in Signal and Image Processing </b>.</li>
        <ul> A. Aldroubi, H. Wang, and <b>K. Zarringhalam.</b> </ul>
	<li>Nonlinear Least Squares in RN, <b>2009, Acta Applicandae Mathematicae </b>.</li>
        <ul> A. Aldroubi and <b>K. Zaringhalam. </b></ul>
	<li>Generating an adaptive multiresolution image analysis with
	  compact cupolets, <b>2007, Journal of Nonlinear Dynamics</b>.</li> 
        <ul><b>K. Zarringhalam</b> and K.Short.</ul>
        </ol>
	<div id="line"></div>
	
        <!--
            <h3>Work in Progress</h3>
	<li>Identification of Biomarker in responders to drug treatment.</li>
	<li>Mapping the ceRNA network of Tumor suppressor genes. </li>
	<div id="line"></div>
	-->  
      </div>
      
      <div id="footer">
	<div id="fl">
	  <p class="links"><a href="index.html">Home</a>
		  
		  <!--<a href="Teaching.html">Teaching</a>-->
		  <a href="Publications.html">Publications</a><a href="Software.html">Software</a>
		  <a href="Contact.html">Contact</a>
		</p>
	</div>
      </div>	
    </div>
  </body>
</html>
